lib/Dialect/Moore/MooreOps.cpp

```diff
@@ -21,6 +23,14 @@ using namespace circt::moore;
 // InstanceOp
 //===----------------------------------------------------------------------===//
 
+LogicalResult InstanceOp::verify() {
```

Comment:
nit: You can verify the same property by just adding `HasParent<"SVModuleOp">` to InstanceOp tablegen definition.

Reply:
Thanks! I'll take it in mind.